ql/src/java/org/apache/hadoop/hive/ql/ddl/table/drop/DropTableAnalyzer.java

```diff
@@ -46,9 +48,13 @@ public DropTableAnalyzer(QueryState queryState) throws SemanticException {
 
   @Override
   public void analyzeInternal(ASTNode root) throws SemanticException {
-    String tableName = getUnescapedName((ASTNode) root.getChild(0));
+    TableName qualTabName = getQualifiedTableName((ASTNode) root.getChild(0));
+    String tableName = qualTabName.getTable();
```

Comment:
`qualTabName.getTable()` -> `qualTabName.getNotEmptyDbTable`

Reply:
Thanks for reviewing! Incorporated the change.